core/src/main/res/values/dimens.xml

```diff
@@ -27,4 +27,7 @@
   <!-- Workaround to store floats in dimen -->
   <item format="float" name="material_light_hint_alpha" type="dimen">0.38</item>
   <item format="float" name="material_dark_hint_pressed_alpha" type="dimen">0.70</item>
+
+  <dimen name="card_margin">5dp</dimen>
+  <dimen name="card_album_radius">0dp</dimen>
```

Comment:
Is this used? static analysis is complaining but I haven't checked them yet. I imagine githooks are being bypassed?